src/index.d.ts

```diff
@@ -0,0 +1,121 @@
+declare module 'trello' {
```

Comment:
The trello modules does not provide any types?

Reply:
Apparently, it doesn't. I searched on npm for @types/trello and found just an open-source initiative, which the last update was 4 years ago. 

Executed the command npm i --save-dev @types/trello, and got the following error:
```
404  '@types/trello@*' is not in this registry.
```
Therefore, searching through the Trello SDK repo issues, I found [one](https://github.com/norberteder/trello/issues/59) thread with some basic types for the package we are using. And based on that I create this index.d.ts file. 

If I missed a better approach to handle this case, please let me know.